domain/access/state/user.go

```diff
@@ -120,11 +122,14 @@ func (st *UserState) GetAllUsers(ctx context.Context) ([]user.User, error) {
 	var usrs []user.User
 	err = db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
 		getAllUsersQuery := `
-SELECT (u.uuid, u.name, u.display_name, u.created_by_uuid, u.created_at, u.last_login, u.disabled) AS (&dbUser.*),
+SELECT (u.uuid, u.name, u.display_name, u.created_by_uuid, u.created_at, u.disabled, ull.last_login) AS (&dbUser.*),
        creator.name AS &dbUser.created_by_name
 FROM   v_user_auth u
        LEFT JOIN user AS creator 
        ON        u.created_by_uuid = creator.uuid
+       LEFT JOIN v_user_last_login AS ull 
```

Comment:
Because this is a left join now what happens when `last_login` is null or empty string. Do we default off to the zero value of time?

Reply:
We do yea. I'm not changing the behavior from before here, though whether its right is a different matter.
IMO the field on user should really be a pointer to `time.Time`. I'll check with the others if we should change this.